spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -90,6 +90,7 @@ https://developers.google.com/identity/protocols/googlescopes[OAuth2 scope] for
 Cloud Spanner credentials | No | https://www.googleapis.com/auth/spanner.data
 | `spring.cloud.gcp.spanner.createInterleavedTableDdlOnDeleteCascade` | If `true`, then schema statements generated by `SpannerSchemaUtils` for tables
 with interleaved parent-child relationships will be "ON DELETE CASCADE". The schema for the tables will be "ON DELETE NO ACTION" if `false`. | No | `true`
+| `spring.cloud.gcp.spanner.transactional-annotation.enabled` | Enables support for Spring Data @Transactional annotation | No | `true`
```

Comment:
I would say, configures `SpannerTransactionaManager` for use with declarative transaction management and `@Transactional`.